build/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax = docker/dockerfile:1.5
-FROM jupyter/pyspark-notebook:spark-3.3.1
+FROM jupyter/pyspark-notebook:spark-3.5.1
```

Comment:
I'm getting this when I try to `docker compose up --build`. Going to downgrade back to 3.3.1 for now.

```
 => ERROR [notebooks internal] load metadata for docker.io/jupyter/pyspar  0.1s
------
 > [notebooks internal] load metadata for docker.io/jupyter/pyspark-notebook:spark-3.5.1:
------
failed to solve: jupyter/pyspark-notebook:spark-3.5.1: docker.io/jupyter/pyspark-notebook:spark-3.5.1: not found
```